Automator/AutoGen.py

```diff
@@ -64,9 +64,20 @@ def __init__(
 
         self.img_dir = img_dir
         if self.img_dir != None:
-            self.img_num_l = len(os.listdir(os.path.join(img_dir, "left")))
```

Comment:
Does this mean that stacking is now required when using AutoGen?

Reply:
Stacking is not required, the script will determine based on the hierarchy of the directory whether the data will be saved in the format for stacking or not.